flows/mpls/mpls.go

```diff
@@ -168,11 +220,14 @@ func New() (lwotg.FlowGeneratorFn, gnmit.Task, error) {
 			klog.Infof("MPLSFlowHandler send function started.")
 
 			buf := gopacket.NewSerializeBuffer()
-			gopacket.SerializeLayers(buf, gopacket.SerializeOptions{}, hdrs...)
+			gopacket.SerializeLayers(buf, gopacket.SerializeOptions{
+				FixLengths:       true,
+				ComputeChecksums: true,
+			}, hdrs...)
 			size := len(buf.Bytes())
 
 			klog.Infof("MPLSFlowHandler Tx interface %s", tx)
-			handle, err := pcap.OpenLive(tx, 9000, true, pcapTimeout)
+			handle, err := pcap.OpenLive(tx, 1500, true, pcapTimeout)
```

Comment:
why is max len not 9000? just curious

Reply:
It could be -- but I had mis-parsed what this does, it just tells `pcap` how many bytes to read from the packet and return, since we never look beyond the headers, we don't actually need 9000 bytes of the packet, so I figured we could set this to less. In one of the other changes, I think I had this as a `const`.